docs/data/catalog/platform/gc-forms/templates.md

```diff
@@ -0,0 +1,104 @@
+# Platform / GC Forms / Templates
+
+### :warning: Note
+This dataset is still in testing and only a snapshot of Staging data is available in Superset. 
+
+---
+
+Dataset providing GC Forms template data.  There are 3 tables as part of this dataset:
+
+- `template`: the templates that are used by GC Forms to render the form users fill out and submit.
+- `templateToUser`: a mapping of templates to the users that created them.
```

Comment:
This mapping is a many to many relationship.  It's not necessarily the user that created the form but a user who is a co-owner of a form.